svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiProps.java

```diff
@@ -49,6 +51,11 @@ public String getClaimRequestUrl(String path) {
     }
     return baseUrl + path;
   }
+
+  public String getAvailabilityUrl() {
```

Comment:
Probably dont need this here if you remove the `ServiceBipApi.isApiFunctioning()` method.

Reply:
I've decided to keep this function, because it helps clarify the use case this URL has. First, I would argue that isApiFunctioning is a valuable troubleshooting element.  For more details about why please see: https://github.com/department-of-veterans-affairs/abd-vro/pull/3280#discussion_r1716059336.  If we accept that isApiFunctioning has value, the original implementation was using the special_issue_types endpoint to validate the request details to the BIP API were functioning properly.  By splitting out a function called getAvailabilityUrl and returning special_issue_types url in that function, it helps formalize the use of that endpoint for this purpose. it is just more explicit to keep the function, and there is no downsides that i can see.